setup.py

```diff
@@ -1,17 +1,18 @@
 #!/usr/bin/env python
 
 from setuptools import setup, find_packages
+
 import provider
 
 setup(
-    name='django-oauth2-provider',
+    name='edx-django-oauth2-provider',
```

Comment:
edx-platform has some uninstall logic that's used during "paver install_prereqs" that could be useful.